stingray/varenergyspectrum.py

```diff
@@ -0,0 +1,176 @@
+from __future__ import division
+import numpy as np
+from stingray.gti import check_separate, cross_two_gtis
+from stingray.lightcurve import Lightcurve
+from stingray.utils import assign_value_if_none, simon
+from stingray.crossspectrum import AveragedCrossspectrum
+import six
+
+
+class VarEnergySpectrum(object):
+    def __init__(self, events, freq_interval, energy_spec, ref_band,
+                 bin_time=1, use_pi=False, log_distr=False,
+                 segment_size=None, events2=None):
+
+        """Generic variability-energy spectrum.
+        
+        Parameters
+        ----------
+        events : stingray.events.EventList object 
+            event list
+        freq_interval : [f0, f1], floats
+            the frequency range over which calculating the variability quantity
+        energy_spec : [emin, emax, N]
+            minimum and maximum energy, and number of intervals, of the final 
+            spectrum
+        ref_band : [emin, emax]
+            minimum and maximum energy of the reference band
+            
+        Other Parameters
+        ----------------
+        use_pi : boolean
```

Comment:
I don't know how this will come out in the documentation, but it might be useful to include the default value in the docstring? I think I've been inconsistent with doing that, but it would be useful to decide on a standard one way or another.